src/CppUTestExt/MockNamedValue.cpp

```diff
@@ -138,6 +144,17 @@ SimpleString MockNamedValue::getType() const
     return type_;
 }
 
+bool MockNamedValue::getBoolValue() const
+{
+    if(type_ == "int")
+        return (value_.intValue_ != 0);
```

Comment:
Usually this happens accidentally, so I'm not sure whether we would want the conversion. What was your thought behind this one?